tools/update_examples.go

```diff
@@ -0,0 +1,306 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/grafana/regexp"
+	"golang.org/x/exp/slices"
+)
+
+var ghToken string
+
+// todo run it by cron & create a PR if needed
+func main() {
```

Comment:
I suggest a make target first.

Reply:
This seems to be recurring theme for my code review :D
:heavy_check_mark:  Added